src/main/java/org/retest/rebazer/service/PullRequestLastUpdateStore.java

```diff
@@ -10,25 +12,27 @@
 @Service
 public class PullRequestLastUpdateStore {
 
-	private final Map<RepositoryConfig, Map<Integer, String>> pullRequestUpdateStates = new HashMap<>();
+	private final Map<RepositoryConfig, Map<Integer, Date>> pullRequestUpdateStates = new HashMap<>();
+	public static final String DEFAULT_REPOSITORY_TIME = "2019-01-01T00:00:00Z";
```

Comment:
* Use Date object as constant
* Use FALLBACK as name instead of DEFAULT
* Use `new Date(0)` to create the object with the same value

Reply:
I used the suggested method to convert the string, because new date is deprecated and LocalDate makes things more cumbersome.